pkgs/top-level/all-packages.nix

```diff
@@ -9013,7 +9013,15 @@ with pkgs;
 
   qprint = callPackage ../tools/text/qprint { };
 
-  qscintilla = callPackage ../development/libraries/qscintilla { };
+  qscintilla = if stdenv.isDarwin then
```

Comment:
This can now migrate to `libsForQt5.qscintilla`

Reply:
Thanks -- migrated accordingly.